public/assets/main.css

```diff
@@ -159,6 +172,36 @@ nav#game_menu {
   font-weight: bold;
 }
 
+nav .submenu {
```

Comment:
why did you add so much css

Reply:
tl;dr because it’s easier, flexible and most of it plain necessary for the menus to function the way they should.

100 – 108: consolidation of nav menus (cf. changes to game_page). This could be moved back inline and added to navigation – would result in more and duplicated code. (… and of course even more so, if more menus would get implemented down the road.)
175 – 183 only apply if items are rendered in a submenu. games_link and menu_item are used for items in menu as well as submenu, adding props with this differentiation there wouldn’t work as easy as adding a few lines css.
185 – 200 is due to the menu differences depending on screen size. What’s displayed at > 600px gets moved into a submenu on <= 600px, hence the media queries.
202 – 204 looks like a vestige from concurrent profile review. I’ll check.

Some screenshots with opened menus:

600px logged in:
![image](https://user-images.githubusercontent.com/33390595/107848300-9688e800-6df2-11eb-9a99-245c878505eb.png)

<= 600px logged in:
![image](https://user-images.githubusercontent.com/33390595/107848302-a274aa00-6df2-11eb-9ac3-76d0bf9e99d7.png)

<= 600px logged in:
![image](https://user-images.githubusercontent.com/33390595/107848311-bddfb500-6df2-11eb-95f4-d93366ca2013.png)

320px logged out:
![image](https://user-images.githubusercontent.com/33390595/107848315-c46e2c80-6df2-11eb-88ef-15030d2fc56c.png)

320px logged in, additional menu rendered over game menu:
![image](https://user-images.githubusercontent.com/33390595/107848343-f1bada80-6df2-11eb-961e-1b961b7c9368.png)